frontend/views/containers/contributions/NonMonetaryPledges.vue

```diff
@@ -0,0 +1,174 @@
+<template lang='pug'>
+fieldset(
+  data-test='nonMonetaryPledges'
+  v-error:pledges='{ attrs: { "data-test": "badPledges" } }'
+)
+  legend.has-text-bold.c-legend
+    i18n.is-title-4 Non-monetary pledge
+    i18n.c-optional(v-if='optional') (optional)
```

Comment:
Note: typically I'd request that these be made two complete sentence using a computed property, however, I see that in the UI they are visually separate elements, so this is OK.